src/sentry/tasks/reports.py

```diff
@@ -749,7 +749,7 @@ class DistributionType(NamedTuple):
 
 def build_project_breakdown_series(reports):
     def get_legend_data(report):
-        accepted_errors, accepted_transactions, filtered, rate_limited = report.series_outcomes
+        accepted_transactions, accepted_errors, filtered, rate_limited = report.series_outcomes
```

Comment:
It would probably make more sense to swap the order that we return from here https://github.com/getsentry/sentry/blob/3a068eb121acaab030e2b6fc856247038d82ecf0/src/sentry/tasks/reports.py#L357-L370